adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -141,6 +142,14 @@
     public static final String CHECKOUT_SHOPPER_HOST_TEST = "checkoutshopper-test.adyen.com";
     public static final String CHECKOUT_SHOPPER_HOST_LIVE = "checkoutshopper-live.adyen.com";
 
+    public static final Set<String> HPP_RESPONSE_PARAMETERS = new HashSet<>(Arrays.asList(HPPConstants.Response.MERCHANT_REFERENCE,
+                                                                                          HPPConstants.Response.SKIN_CODE,
+                                                                                          HPPConstants.Response.SHOPPER_LOCALE,
+                                                                                          HPPConstants.Response.PAYMENT_METHOD,
+                                                                                          HPPConstants.Response.AUTH_RESULT,
+                                                                                          HPPConstants.Response.PSP_REFERENCE,
+                                                                                          HPPConstants.Response.MERCHANT_RETURN_DATA));
+
```

Comment:
why making this a static list ? This can cause issues if we responds back with a new parameter that is not in the list. We had the same issue on SFCC. We should make this dynamic as signature is always constructed on all the fields that are provided back

Reply:
Adyen platform uses only these fields when calculating a merchant signature.

Reply:
What is the benefit of static vs dynamic ? You make it dynamically for for additionalData.